### DISCUSSION TITLE
Lost initial diagram before signup

### DISCUSSION THREAD

**Discussion by jossh-h** [excalidraw/excalidraw#6535] [Q&A]
I as testing https://excalidraw.com/ and had an initial digram I had spent a fair amount of time working on, happy with the features of excalidraw I decided to sign up.
After signup I refreshed the page and I have lost my initial work ;(

Also every time I try to go to excalidraw.com it just asks me to sign in..

Is there any way to get that back?
I have tried to look at my browser history but for some reason its not there..

Safari-16.2
Ventura-13.1

**Comment by sojusnik** [comment]
I feel with you, you are not alone, see https://github.com/excalidraw/excalidraw/discussions/6463#discussioncomment-5634149
